docs/Microservices/overview.md

```diff
@@ -84,6 +84,27 @@ For more information about power states supported by the REST APIs, see [Intel®
 
 Intel® AMT enables remote management of a device, even when the OS isn't running, through KVM over IP support. No additional equipment is needed for this feature.  With KVM control, IT administrators can access and update PCs and devices as if they were onsite. It eliminates the need for remote KVM switches and other hardware. 
 
+## Passwords
+
+There are four levels of passwords: 
+
+1. **Intel® Manageability Engine BIOS Extensions (MEBX) Password : MEBX Password - ** Use this password to secure the local Intel® MEBX menu. This password is only used when physically accessing the managed device during system boot. Access the menu with Ctrl-P on most devices.
+2. **Sample Web UI Password: MPS_WEB_ADMIN_PASSWORD - ** Use this password on the development system when logging into the Sample Web UI. The Sample Web UI Password uses this default MPS user authentication credential when it triggers MPS to issue a JWT (JSON Web Token). In most production environments, this default credential is replaced by a more rigorous authentication protocol. Examples include OAuth2, OpenID, or an authentication service that can issue an Auth Token to be validated by the API gateway. 
+3. ** ACM & CCM Profiles: AMT Password - ** This CIRA credential is used by Intel® AMT managed devices to authenticate the MPS when establishing the CIRA connection. Groups of devices can use the same password, but a unique password per device is recommended. Intel® AMT verifies this password when it gets a command from the MPS server.  
```

Comment:
Looks like the AMT Password got mixed together with the MPS CIRA credentials.
AMT Password - this is the remote admin password for AMT that is set in the ACM and CCM profiles.  This is the password that the MPS uses to authenticate itself with AMT when issuing a command to have AMT do something (power control, KVM, etc).  Recommended that this password be unique per device
MPS credentials - this is the username and password that AMT uses when establishing a CIRA connection with MPS.  MPS verifies this credential against what it has stored in Vault for a given system.  Recommended that this password is unique per device.

Reply:
No, that isn't what happened. That is what BW suggested. Correction. It's how I interpreted BW's explanation, and my interpretation could be wrong. Decide (amongst yourselves) if you want me to go with your explanation.

Reply:
Ok, please split these per my suggestion.  These are two different credentials.